crates/holochain/src/conductor/conductor.rs

```diff
@@ -311,16 +314,26 @@ where
                 .map_err(Box::new)?;
         // TODO: RELIABILITY: Handle this task by restarting it if it fails and log the error
         self.manage_task(ManagedTaskAdd::dont_handle(task)).await?;
-        self.app_interface_signal_broadcasters
-            .insert(interface_id, signal_broadcaster);
+        self.signal_broadcasters
+            .push((Some(interface_id), signal_broadcaster));
         Ok(port)
     }
 
+    pub(super) fn add_signal_channel(
+        &mut self,
+    ) -> ConductorResult<tokio::sync::broadcast::Receiver<Signal>> {
+        let (signal_broadcaster, signal_receiver) =
+            tokio::sync::broadcast::channel(SIGNAL_BUFFER_SIZE);
+        // no associated AppInterfaceId
+        self.signal_broadcasters.push((None, signal_broadcaster));
+        Ok(signal_receiver)
```

Comment:
return the signal_receiver so that the caller can listen to the signals stream, which is prbly what they want to do (I did)